bb-worker/Dockerfile.apt.python3.7

```diff
@@ -0,0 +1,42 @@
+ARG FROMBASEOS
```

Comment:
We have FROMBASEOS declared at the top of this file, and BASEOS delclared farther down. But in all the places those arguments are passed, they are identical to one another. Should these two arguments be combined into one?

Reply:
This was the only way I could get it to work.

Although I could find no documentation supporting my experience, when a build arg is build in the FROM line **it seems to be consumed, and not available for use later in the build**

Try it yourself! Originally I have the same `BASEOS` var used through out the Dockerfile, since they are the same value. But the var would be render empty, "", anywhere after the FROM line.

So, I made 2, equal vars :man_shrugging: . Not ideal, I'm open to alternative solutions!

Reply:
huh, ok then :)